### ISSUE TITLE
RelationIds set to true generates incorrect types

### ISSUE THREAD

**Issue by ebrahimB** [Kononnable/typeorm-model-generator#207]
I have two tables: `user` and `role`. The `user` table has a column called `role_id` of type `INT`. It also has a foreign key referencing the `id` column in the `role` table. 

Using model generator with the following options:
```
typeorm-model-generator -d db_name -h $dbhost -u $user -x $password -p 3306 -e mysql -o "$output" --cf "camel" --ce "pascal" --cp "camel" --noConfig --relationIds true
```
yields the following in the `user` model file: 

```
    @ManyToOne(()=>Role, (role: Role)=>role.users,{  nullable:false,onDelete: 'NO ACTION',onUpdate: 'NO ACTION' })
    @JoinColumn({ name:'role_id'})
    role:Role | null;

    @RelationId((user: User) => user.role)
    roleId: number[];
```
The issue is that the generated `roleId` property should be of type `number` and not `number[]`.

**Comment by natiels** [comment]
entity.mst line 37 needs to be changed to this.
```
{{toPropertyName ../tsName}}Id: {{#if (or isOneToMany isManyToMany)}}{{toLazy (concat  ../tsType "[]")}}{{else}}{{toLazy ../tsType}}{{/if}};{{/if}}{{/relations}}
```
It should use the same logic as is used for the @JoinColumn.  I just copied the "or" condition from line 31 and used it on 37 to determine if "[]" should be added.

**Comment by Kononnable** [comment]
Implemented in `typeorm-model-generator@0.4.3`
